providers-sdk/v1/resources/schema.go

```diff
@@ -3,15 +3,21 @@
 
 package resources
 
+type ResourcesSchema interface {
```

Comment:
Since we are moving this under the `resources` namespace, could we name this `Schema` instead?

Reply:
There is already `Schema` struct in the proto. That's why I had to give the interface a different name, otherwise it clashes with the struct name

Reply:
Gotcha, let's clean it up in a follow-up and get merged first :)